homeassistant/components/tado/climate.py

```diff
@@ -464,16 +494,24 @@ def _control_hvac(self, hvac_mode=None, target_temp=None, fan_mode=None):
             # A temperature cannot be passed with these modes
             temperature_to_send = None
 
+        fan_speed = (
+            self._current_tado_fan_speed
+            if (self._support_flags & SUPPORT_FAN_MODE)
+            else None
+        )
+        swing = (
+            self._current_tado_swing_mode
+            if (self._support_flags & SUPPORT_SWING_MODE)
+            else None
+        )
```

Comment:
We should avoid inline if-statements if they span multiple lines. They are not very readable. You're actually using 1 more row than if you would do a normal if…else statement.

Reply:
Will adjust

Reply:
done